### DISCUSSION TITLE
how do I configure nx-console to work on ubuntu - every command fails because it does not recognise them

### DISCUSSION THREAD

**Discussion by DustinAPI** [nrwl/nx-console#1571] [Q&A]
I assume it is because it is not using my user's .bashrc or it is not running at the workspace root?

" *  Executing task: npx nx build md-api 

/usr/bin/bash: line 1: npx: command not found

 *  The terminal process "/usr/bin/bash '-c', 'npx nx build md-api'" terminated with exit code: 127. 
 *  Terminal will be reused by tasks, press any key to close it. 
"

**Comment by mklueh** [comment]
Same here. 

Could you solve it reliably?
